transmission-daemon: add page (#8457)
tldr-pages/tldr: pages/common/transmission-daemon.md

# transmission-daemon

> Daemon controlled with `transmission-remote` or its web interface.
> See also: `transmission`.
> More information: <https://manned.org/transmission-daemon>.

> - Start a headless Transmission session:

Reviewer: ```suggestion
- Start a headless `Transmission` session:
```

Author: I guess since `Transmission` isn't actually a command to run anything, I was just treating it as more of its name. If you think this is an important distinction, then maybe it should just be this?
```suggestion
- Start a headless `transmission` session:
```

Reviewer: Yes, I agree. I don't think it should be capitalized.

`transmission-daemon`

> - Specify a directory to watch for new torrent files:

Reviewer 2: ```suggestion
- Start and watch a specific directory for new torrent files:
```

> - Start and watch a specific directory for new torrent files:

Reviewer: ```suggestion
- Start and watch a specific directory for new torrents:
```

What do you think of this change, I am fine with either.

`transmission-daemon --watch-dir {{path/to/directory}}`

> - Generate a configuration file for the daemon:

Reviewer 2: ```suggestion
- Dump daemon settings in JSON format:
```

`transmission-daemon --dump-settings > {{path/to/file.json}}`

> - Run the daemon with explicit settings for the web interface:

Reviewer 2: ```suggestion
- Start with specific settings for the web interface:
```

> `transmission-daemon --auth --username {{username}} --password {{password}} --port {{9091}} --allowed "{{127.0.0.1}}"`

Reviewer 2: IPs or hotnames don't need quotation marks :thinking: 
```suggestion
`transmission-daemon --auth --username {{username}} --password {{password}} --port {{9091}} --allowed {{127.0.0.1}}`
```

Author: Oh, I guess you're right. I borrowed this from the Arch Wiki, and I assume they put quotes since it's actually specifying a string where a list of IPs can be included. Not really necessary here though.
